sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_patch.py

```diff
@@ -0,0 +1,985 @@
+    :ivar version: Version of schema.
+    :vartype version: int
+    """
+
+    def __init__(self, **kwargs: Any) -> None:
+        self.id = kwargs.pop("id")
+        self.format = kwargs.pop("format")
+        self.group_name = kwargs.pop("group_name")
+        self.name = kwargs.pop("name")
+        self.version = kwargs.pop("version")
+
+    def __repr__(self):
+        return (
+            f"SchemaProperties(id={self.id}, format={self.format}, "
+            f"group_name={self.group_name}, name={self.name}, version={self.version})"[
+                :1024
+            ]
+        )
+
+
+class Schema(object):
+    """
+    The schema content of a schema, along with id and meta properties.
+
+    :ivar definition: The content of the schema.
+    :vartype definition: str
+    :ivar properties: The properties of the schema.
+    :vartype properties: SchemaProperties
+    """
+
+    def __init__(self, **kwargs: Any) -> None:
+        self.definition = kwargs.pop("definition")
+        self.properties = kwargs.pop("properties")
+
+    def __repr__(self):
+        return f"Schema(definition={self.definition}, properties={self.properties})"[
+            :1024
+        ]
+
+
+class SchemaFormat(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    """
+    Represents the format of the schema to be stored by the Schema Registry service.
+    """
+
+    AVRO = "Avro"
+    """Represents the Apache Avro schema format."""
+
+    JSON = "Json"
+    """Represents the JSON schema format."""
+
+    CUSTOM = "Custom"
+    """Represents a custom schema format."""
+
+
+class ApiVersion(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    """
+    Represents the Schema Registry API version to use for requests.
+    """
+
+    V2021_10 = "2021-10"
+    V2022_10 = "2022-10"
+    """This is the default version."""
+
+
+DEFAULT_VERSION = ApiVersion.V2022_10
+
+
+###### Encoder Protocols ######
+
+
+class SchemaContentValidate(Protocol):
+    # TODO: make __call__ a public API so that docs show, until then, keep below docstring
+
+    """
+    Callable protocol which validates content against provided schema. If invalid, raises Exception.
+     Else, returns None.
+
+    :param mapping[str, any] schema: The schema to validate against.
+    :param mapping[str, any] content: The content to validate.
+
+    :rtype: None
+    :returns: None if valid.
+    :raises: Exception if content is invalid against provided schema.
+    """
+
+    def __call__(self, schema: Mapping[str, Any], content: Mapping[str, Any]) -> None:
+        """
+        Validates content against provided schema. If invalid, raises Exception.
+         Else, returns None.
+
+        :param mapping[str, any] schema: The schema to validate against.
+        :param mapping[str, any] content: The content to validate.
+
+        :rtype: None
+        :returns: None if valid.
+        :raises: Exception if content is invalid against provided schema.
+        """
+
+
+class MessageContent(TypedDict):
```

Comment:
The type of this in ApiView changed to ForwardRef(bytes/str)

Reply:
no that was unintentional, will fix!

Reply:
so I copied this over exactly as it was. I think it could be that something changed with the apiview generator? will look into it...

Reply:
Travis is not sure either, but says maybe it's the Python version update. (Not exactly sure why that would be causing this though.)